docs/how_we_work/issue_review.md

```diff
@@ -0,0 +1,12 @@
+# Reviews Process for Issues
+
+As a team we will use labels for tracking which reviews are assign to whom and issue comments to tag our teammates when an issue is ready for their review.
+
+## Process
+
+- When an issue is ready for review, move to "In Review"
+- Add `____ Review` label for those who need to review and also comment, "@" tagging them on the issue stating that it's ready for their review
+  - If the issue has a corresponding PR, also assign those who need to review the PR as "Reviewers" on the PR
+  - If reviews need to happen sequentially, add just the label for the first review, then when the first review is done, remove the first and add second label and so on
+
+Tip: to see all open issues with your "Review" label go to: [this link](https://github.com/18F/OPRE-Unicorn/issues?q=is%3Aopen+is%3Aissue) and  select the "Labels" dropdown and click the label that corresponds with your name. You can save this search as a bookmark for one-click access to what issues need your review.
```

Comment:
Love the link, handy! I'm noticing that the link is hard-coded to the 18F GitHub repo, I wonder if it would work as a relative link since we're planning to transfer to a different Org this week?

Reply:
I can sure try!

Reply:
Ok I tried and it doesn't work as relative, because then it tries to do some funky stuff by injecting the branch name, etc into the path, like so: https://github.com/18F/OPRE-Unicorn/blob/cj/31/review-process/issues?q=is%3Aopen+is%3Aissue. LMK if you have ideas on how to make it not do that! Maybe this would not be the case when it was merged into `main`?

Reply:
Possibly! This isn't a blocker, just something I wanted to raise as a question